### PR TITLE
Redirect ex002 to docs.gmt-china.org

### PR DESCRIPTION
**Pull request by seisman**
https://gmt-china.org/example/ex002/ will be redirected to https://docs.gmt-china.org/latest/examples/ex002/ in **0** seconds.

The time interval can be changed to other values.

Please give it a try.

### CODE REVIEW

File: layouts/partials/head_custom.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{{ with .Params.redirect }}
+<meta http-equiv="refresh" content="0; URL={{ . }}" />
```
Comment (core-man) [inline layouts/partials/head_custom.html:2]: 0 means time interval?

Comment (seisman) [reply layouts/partials/head_custom.html:2]: Yes.

**Comment by seisman** [comment]
You can visit https://gmt-china.github.io/sitepreview/gmt-china/gmt-china.org/redirect/example/ex002/ to preview how it works.

**Comment by core-man** [comment]
> You can visit https://gmt-china.github.io/sitepreview/gmt-china/gmt-china.org/redirect/example/ex002/ to preview how it works.

Looks good.

**Comment by core-man** [comment]
Do we want to keep the blog?https://gmt-china.org/blog/china-border-issues/

**Comment by seisman** [comment]
This PR is an example showing how to redirect a page. We can decide if a page should be redirected or not in separate PRs.

**Review by core-man — APPROVED** [review]

**Pull request merged**